cd/build_images.sh

```diff
@@ -14,7 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -e
+# -a: Always export variables to subsequent commands.
+# Required for variables from sourced env file to automatically be visible to docker compose.
+set -e -a
```

Comment:
Does this also need to happen in `publish_images.sh`? Or even everything in the `cd` directory?

Reply:
No. In `publish_images.sh` the variables are only used within the script and not by any child processes. Same for the others. It's just `docker compose` which uses/needs them, and only since the change to use legacy images.